pages/getting-started/install-vanilla.mdx

```diff
@@ -0,0 +1,32 @@
+# Set up our full Starter Kit
+
+Now that you've finished the homework and have a good idea of how Embeddable components work, you're all set to build whatever components you want. That said, you can also make use of our existing library of charts and controls instead of having to reinvent the wheel. This starter library comes as a single NPM package (it's already in the boilerplate you downloaded earlier) and has full support for everything Embeddable does, including [using themes](/theming/introduction).
+
+Here's what you need to do to get set up:
+
+<Steps>
+### Delete the Example components
+
+You probably don't need a "Hello World" component or a "TextInput" component, and our starter kit includes a more robust Pie chart. So go ahead and delete the entire `example` directory in the `src/embeddable.com/components` directory.
+
+### Uncomment the starter kit
+
+Open `embeddable.config.ts` and uncomment the `componentLibraries` line (typically line 14 if you haven't changed anything else). Save that file.
```

Comment:
@cwbuecheler where you say `line 14` you could link that to line 14 in the repo, so it's easy for them to see what you mean if they're not sue